scripts/deploy.sh

```diff
@@ -13,7 +13,17 @@ key="$2"
 public_port=80
 
 scp -o StrictHostKeyChecking=no -i "$key" scripts/remote-deploy.sh "$ssh_hostname":/tmp/remote-deploy.sh
-revision_browser=$(scripts/latest-revision.sh https://github.com/libero/browser.git)
-revision_dummy_api=$(scripts/latest-revision.sh https://github.com/libero/dummy-api.git)
-revision_pattern_library=$(scripts/latest-revision.sh https://github.com/libero/pattern-library.git)
-ssh -o StrictHostKeyChecking=no -i "$key" "$ssh_hostname" PUBLIC_PORT=${public_port} REVISION_BROWSER="$revision_browser" REVISION_DUMMY_API="$revision_dummy_api" REVISION_PATTERN_LIBRARY="$revision_pattern_library" /tmp/remote-deploy.sh
+
+# list of applications
+declare -a applications=(browser dummy-api pattern-library)
```

Comment:
Could even move this list to separate file and share it between the two scripts?

Reply:
One script is uploaded to the server, so it cannot include external files and needs to receive everything as a variable.